src/data/quot.lean

```diff
@@ -77,6 +78,9 @@ noncomputable def quotient.out [s : setoid α] : quotient s → α := quot.out
 
 @[simp] theorem quotient.out_eq [s : setoid α] (q : quotient s) : ⟦q.out⟧ = q := q.out_eq
 
+theorem quotient.out_eq_iff [s : setoid α] (q : quotient s) (x : α) : ⟦x⟧ = q ↔ x ≈ q.out :=
```

Comment:
`git grep mk_eq_iff` gives no matches. Or do you mean that I should rename this theorem? I don't like `mk_eq_iff`, because it doesn't mention `out`. Maybe, `mk_eq_iff_eqv_out`?